Resources/Locale/en-US/_NF/crayon/crayon-component.ftl

```diff
@@ -0,0 +1,7 @@
+
+## Entity
+
+crayon-drawing-label-unlimited = Drawing: [color={$color}]{$state}[/color]
+
+## UI
+crayon-window-title = Crayon
```

Comment:
This comes from upstream already, so it's duplicated. YAML Linter is screaming at this existing

Reply:
What do you suggest I rename it to? I need that so that the UI is not cluttered up w/ the ridiculous maxinteger

Reply:
Nuke that line. That locale is already coming from the upstream `crayon-component.ftl`